sui/utils/LedgerSigner.js

```diff
@@ -41,7 +41,7 @@ class LedgerSigner extends Signer {
             await this.init();
         }
 
-        return await this.sui.getPublicKey(this.path);
+        return this.sui.getPublicKey(this.path);
```

Comment:
removing these `await`s seems incorrect. It should also be done in a separate PR